src/devices/ft004b.c

```diff
@@ -0,0 +1,77 @@
+#include "rtl_433.h"
+#include "data.h"
+#include "util.h"
+
+static uint8_t
+reverse_bits(uint8_t value)
```

Comment:
There is a `uint8_t reverse8(uint8_t x)` in util.h. I frequently miss that function too, assuming it would be in bitbuffer.h.